code/src/GridAPI/Rows.ts

```diff
@@ -58,7 +58,7 @@ namespace GridAPI.Rows {
      * @param {string} gridID ID of the Grid where the change will occur.
      * @returns {*}  {string} Resulting code and message in JSON format
      */
-    export function AddRows(gridID: string): string {
+    export function AddRows(gridID: string, numberOfRows: number): string {
```

Comment:
Shouldn't we have a default on numberOfRows = 1?

Reply:
Done